documentation/endpoints/posts.md

```diff
@@ -1,91 +1,9 @@
 ## Posts
 
-## Create a Post and/or Create/Update a Signup
```

Comment:
I think we still want to document how to create a post for this endpoint because we do still use it

Reply:
@katiecrane sorry this is a little misleading! It is still here but I just moved it to under the GET endpoint documentation. I pinged you the line where it was moved to! :)